terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars

```diff
@@ -2,8 +2,7 @@ environment = "cidev"
 aws_profile = "development-eu-west-2"
 
 # service configs
-use_set_environment_files = true
-log_level = "debug"
+log_level = "info"
```

Comment:
```suggestion

```
Since the log_level is defined in the [S3](https://github.com/companieshouse/ecs-service-configs-dev/blob/main/development-eu-west-2/cidev/confirmation-statement-web.env) .env file, the log_level from these profile/vars can be removed. Unless you've been asked and/or the value isn't being pulled in correctly at runtime?